tests/php/tests/includes/test_class.wp-job-manager-data-exporter.php

```diff
@@ -0,0 +1,194 @@
+	}
+
+	/**
+	 * The data provider method
+	 *
+	 * @return array
+	 */
+	public function data_provider() {
+		return array(
+			array(
+				array(
+					'_company_logo'    => 'https://example.com/company/logo',
+					'_company_name'    => 'Example',
+					'_company_website' => 'https://example.com/',
+					'_company_tagline' => 'Just another tagline',
+					'_company_twitter' => 'https://twitter.com/example?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor',
+					'_company_video'   => 'https://example.com/company/video',
+				),
+				array(
+					'data' => array(
+						'group_id'    => 'wpjm-user-data',
+						'group_label' => __( 'WP Job Manager User Data' ),
+						'item_id'     => '', // the item_id depends on the ID of the user.
+						'data'        => array(
+							'Company Logo'    => array(
+								'name'  => 'Label',
+								'value' => true, // specify that attachment should be created.
+							),
+							'Company Name'    => array(
+								'name'  => 'Label',
+								'value' => 'Example',
+							),
+							'Company Website' => array(
+								'name'  => 'Label',
+								'value' => 'https://example.com/',
+							),
+							'Company Tagline' => array(
+								'name'  => 'Label',
+								'value' => 'Just another tagline',
+							),
+							'Company Twitter' => array(
+								'name'  => 'Label',
+								'value' => 'https://twitter.com/example?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor',
+							),
+							'Company Video'   => array(
+								'name'  => 'Label',
+								'value' => 'https://example.com/company/video',
+							),
+						),
+					),
+					'done' => true,
+				),
+			), // End of first set of parameters.
+			array(
+				array(
+					'_company_name'    => 'Example',
+					'_company_website' => 'https://example.com/',
+					'_company_tagline' => 'Just another tagline',
+				),
+				array(
+					'data' => array(
+						'group_id'    => 'wpjm-user-data',
+						'group_label' => __( 'WP Job Manager User Data' ),
+						'data'        => array(
+							'Company Name'    => array(
+								'name'  => 'Label',
+								'value' => 'Example',
+							),
+							'Company Website' => array(
+								'name'  => 'Label',
+								'value' => 'https://example.com/',
+							),
+							'Company Tagline' => array(
+								'name'  => 'Label',
+								'value' => 'Just another tagline',
+							),
+						),
+					),
+					'done' => true,
+				),
+			), // End of second set of parameters.
+			array(
+				array(
+					'_company_logo',
+					'_company_name',
+					'_company_website',
+					'_company_tagline',
+					'_company_twitter',
+					'_company_video',
+				),
+				array(
+					'data' => array(
+						'group_id'    => 'wpjm-user-data',
+						'group_label' => __( 'WP Job Manager User Data' ),
+						'item_id'     => '', // the item_id depends on the ID of the user.
+						'data'        => array(),
+					),
+					'done' => true,
+				),
+			), // End of third set of parameters.
+		);
+	}
+}
```

Comment:
From PHPCS:
> File must end with a newline character

Reply:
I don't know why GH isn't tracking it but both the files have an empty line at the end when I open them locally